apps/ui/src/networks/offchain/api/queries.ts

```diff
@@ -360,3 +360,18 @@ export const STRATEGY_QUERY = gql`
   }
   ${STRATEGY_FRAGMENT}
 `;
+
+export const STATEMENTS_AND_USERS_QUERY = gql`
+  query ($where: StatementsWhere, $userIds: [String!]) {
```

Comment:
```suggestion
  query ($statementsWhere: StatementsWhere, $userIds: [String!]) {
```

This way it's clearer when we execute query as to what `where` refers.